### ISSUE TITLE
Add an option to determine directionality of animations

### ISSUE THREAD

**Issue by LindaJuffermans** [ReplayMod/ReplayMod#367]
Two camera keyframes with yaw 596 and yaw 525 sometimes causes the camera to rotate 289 degrees instead of 71.
Maybe an option to indicate if the animation to the next keyframe should be + or - can help in these scenarios?

![keyframe1](https://user-images.githubusercontent.com/31929183/93706933-33d53e00-fb2b-11ea-987b-475812e5be7a.png)
![keyframe2](https://user-images.githubusercontent.com/31929183/93706936-39328880-fb2b-11ea-87ec-68fde3bb2777.png)

Animation: https://www.youtube.com/watch?v=JQAI3-OgeGA

**Comment by Johni0702** [comment]
It should really just always use the shorter way, shouldn't it? At least I think that's what it's supposed to do (and failing at) atm.

**Comment by LindaJuffermans** [comment]
There is really only one path from 596 to 525 (via 595, 594, 593 ... 528, 527, 526)... there is no shorter/longer unless you modulo 360 it.

**Comment by Johni0702** [comment]
> unless you modulo 360 it

Yeah, that's what I was thinking about and I believe we already try to do that.
That seems easier for people to understand than having to keep track of their total yaw (which isn't displayed anywhere except in the Edit Keyframe gui) during editing.

**Comment by LindaJuffermans** [comment]
That may be true but then maybe it can be an option so others who don't want to rely on automation (which may not do what they want/expect) can just rely on a logical path from X to Y?

**Comment by Johni0702** [comment]
Alright, this is however then a feature separate from the bug in the original replay. I've split off the bug in the current interpolator (which is intended to wrap around) into #368.
